### PR TITLE
Fixed path to appversions.json

### PR DESCRIPTION
**Pull request by wagnerand**
...when the executable is called from another directory.

@kumar303 r?

### CODE REVIEW

File: validator/validate.py [hunk 1]
```diff
@@ -58,6 +57,10 @@ def validate(path, format='json',
                          overrides=overrides, for_appversions=for_appversions)
     bundle.save_resource('is_compat_test', compat_test)
 
+    if (approved_applications is None):
```
Comment (mstriemer) [inline validator/validate.py]: No need for the parens here.

File: validator/validate.py [hunk 2]
```diff
@@ -58,6 +57,10 @@ def validate(path, format='json',
...
+        approved_applications = os.path.join(os.path.dirname(__file__),
```
Comment (mstriemer) [inline validator/validate.py]: This looks like it doesn't pass PEP8 but I don't see a build failure. Maybe we aren't running PEP8 tests...

This should be:
```py
approved_applications = os.path.join(
    os.path.dirname(__file__),
    'app_versions.json')
# or
approved_applications = os.path.join(os.path.dirname(__file__),
                                     'app_versions.json')

**Comment by mstriemer** [comment]
r+wc

**Comment by wagnerand** [comment]
All done, thanks.

**Pull request merged**